### DISCUSSION TITLE
TAPO H100

### DISCUSSION THREAD

**Discussion by HaniAlnaji** [petretiandrea/home-assistant-tapo-p100#434] [Q&A]
I installed the TAPO in home assistant and I did the configuration , I put the H100 hub IP with my TPlink email and password , but it is give me failed to setup 

<img width="1072" alt="Screenshot 2023-07-11 at 8 04 49 PM" src="https://github.com/petretiandrea/home-assistant-tapo-p100/assets/89660014/358c92b5-73e7-4181-90fb-6f7b9bc89420">

**Comment by petretiandrea** [comment]
v2.9.1 now fully support the hub

**Reply by HaniAlnaji** [reply]
Thank you appreciate your efforts
